### PR TITLE
Add url_path_tag to openapi operation tags. Use path without prefix with leading / as url path in specification

### PR DESCRIPTION
**Pull request by EvgenySmekalin**
1. Add url_path_tag to openapi operation tags. 
2. Use path without prefix with leading / as url path in specification

### CODE REVIEW

File: winter_openapi/generator.py [hunk 1]
```diff
@@ -113,6 +120,36 @@ def __str__(self):
...
+    if len(url_path_segments) <= len(path_prefix_segments):
+        if len(url_path_segments) == len(path_prefix_segments):
+            return ''
+        else:
+            raise ValueError(f'Invalid path prefix {path_prefix} for url_path {route.url_path}')
+
+    url_path_tag = url_path_segments[len(path_prefix_segments)]
+
+    if url_path_tag.startswith('{'):
+        return ''
```
Comment (mofr) [inline winter_openapi/generator.py:148]: What is the branch for?

Comment (mofr) [inline winter_openapi/generator.py:148]: Probably returning None would be more appropriate

File: winter_openapi/generator.py [hunk 2]
```diff
@@ -51,23 +51,30 @@ def generate_openapi(
...
-def _get_openapi_path(*, route: Iterable[Route], operation_ids: Set[str], tag_names: Iterable[str]) -> PathItem:
+def _get_openapi_path(*, routes: Iterable[Route], operation_ids: Set[str], tag_names: Iterable[str], path_prefix: str) -> PathItem:
     path = {}
-    for route_item in route:
+    for route_item in routes:
         operation_id = route_item.method.full_name
         if operation_id in operation_ids:
             warnings.warn(f"Duplicate Operation ID {operation_id}")
         operation_ids.add(operation_id)
+        url_path_tag = get_url_path_tag(route_item, path_prefix)
```
Comment (mofr) [inline winter_openapi/generator.py:74]: Maybe pass already calculated tag to _get_openapi_path method? instead of the path_prefix

**Review by mofr — CHANGES_REQUESTED** [review]

File: winter_openapi/generator.py (hunk 1 above)
Comment (EvgenySmekalin) [reply winter_openapi/generator.py:149]: For url: `prefix/{id}/get-resource` and prefix `/prefix`

File: winter_openapi/generator.py (hunk 2 above)
Comment (EvgenySmekalin) [reply winter_openapi/generator.py:74]: Do you mean this because it is a group of routes?

Comment (mofr) [reply winter_openapi/generator.py:74]: Yes, group of routes with the same url. Also it will remove the argument and make the code simpler.

**Review by mofr — APPROVED** [review]

**Pull request merged**